csharp/src/iceboxnet/Server.cs

```diff
@@ -56,7 +58,9 @@ public static int Main(string[] args)
 
         var initData = new Ice.InitializationData
         {
-            properties = new Ice.Properties(ref args, new Ice.Properties(["IceBox"]))
+            properties = Ice.Util.createProperties(
```

Comment:
A more elegant way to create properties without default properties.

Reply:
What if we can write something like:

```
(properties, args) = Properties.fromArgs(
    args,
    new Ice.PropertiesOptions { optInPrefixes = ImmutableList.Create("IceBox") })
);
```

Reply:
I prefer the `ref args` over the factory method that returns a pair.